backend/dataall/modules/s3_datasets/api/table/queries.py

```diff
@@ -36,3 +37,13 @@
     resolver=preview,
     type=gql.Ref('QueryPreviewResult'),
 )
+
+listTableDataFilters = gql.QueryField(
+    name='listTableDataFilters',
+    args=[
+        gql.Argument(name='tableUri', type=gql.NonNullableType(gql.String)),
+        gql.Argument('filter', DatasetTableFilter),
+    ],
```

Comment:
picky nit: in a recent PR it was pointed out that we are mixing regular args with key-value args, let's try to be consistent